elements/pfe-navigation/src/pfe-navigation.js

```diff
@@ -682,10 +681,31 @@ class PfeNavigation extends PFElement {
 
       // Convert dropdown links into buttons
       const dropdownButton = document.createElement("button");
+      // Select mobile menu toggle button
+      const mobileMenuToggleButton = this.shadowRoot.getElementById("mobile__button");
+      // Get mobile menu toggle button content id for aria-controls setting
+      const mobileMenuToggleContent = this.shadowRoot.querySelector(".pfe-navigation__menu-wrapper");
+      const mobileMenuToggleContentId = mobileMenuToggleContent.getAttribute("id");
+      // Select search toggle
+      const searchButton = this.shadowRoot.querySelector(".pfe-navigation__search-toggle");
+      // Get search content slot id for aria-controls settings
+      const searchContentId = this.shadowRoot
+        .querySelector(".pfe-navigation__search-dropdown-wrapper")
+        .getAttribute("id");
+      // Select All Red Hat Site Switcher toggle
+      const siteSwitcherButton = this.shadowRoot.querySelector(".pfe-navigation__all-red-hat-toggle");
+      // Select All Red Hat content wrapper
+      const siteSwitcherContent = this.shadowRoot.querySelector(".pfe-navigation__all-red-hat-wrapper");
+      // Get Site Switcher content id
+      const siteSwitcherContentId = siteSwitcherContent.getAttribute("id");
+
       // Move over or add important attributes and content
       dropdownButton.setAttribute("class", dropdownLink.getAttribute("class"));
       dropdownButton.classList.add("pfe-navigation__menu-link--has-dropdown");
+      // set aria-expanded to false initially bc they will be closed on page load
       dropdownButton.setAttribute("aria-expanded", "false");
+      searchButton.setAttribute("aria-expanded", "false");
```

Comment:
I _think_ `this._addCloseDropdownAttributes()` would be a better fit than individually setting these things, but double check me...

Reply:
@wesruv see above comment, using that updated version of that function seems to be working well.